api/routers/comment.js

```diff
@@ -2,35 +2,56 @@ const express = require('express');
 const routerComment = express.Router();
 const User = require('../models/user');
 const Videos = require('../models/Videos');
+const Comment = require('../models/Comment');
 
 // Crear un nuevo commentario
 routerComment.post('/comment/:id', async (req, res) => {
```

Comment:
Aca no hace falta el :id param en el URL, ejemplo:

```
routerComment.post('/comment', async (req, res) => {
```

Este es in POST request y el payload que va a esperar es de la forma:

```
{
        "idVideo": "6160b28f9328682655b65c4c",
        "alumno":: "6160b2829328682655b65c05",
        "text": "Contenido del comentario"
}
```

Ese payload lo va a tener q enviar el front.